Fabric.Authorization.API/scripts/tests/Get-AuthorizationFabricInstallerSecret.tests.ps1

```diff
@@ -0,0 +1,72 @@
+param(
+    [string] $targetFilePath = "$PSScriptRoot\..\Install-Authorization-Utilities.psm1"
+)
+
+ Write-Host $targetFilePath
+# Force re-import to pick up latest changes
+Import-Module $targetFilePath -Force
+
+ Describe 'Get-AuthorizationFabricInstallerSecret' {
+    InModuleScope Install-Authorization-Utilities {
+        Context 'Happy Paths' {
+            BeforeAll {
+                $encryptionCertificateThumbprint = "some thumbprint"
+            }
+
+            It 'Should decrypt and return the installer secret' {
+                # Arrange
+                $fabricInstallerSecret = "!!enc!!:some secret"
+                Mock -CommandName  Unprotect-DosInstallerSecret { return $fabricInstallerSecret }
+
+                # Act
+                $decryptedSecret = Get-AuthorizationFabricInstallerSecret `
+                    -fabricInstallerSecret $fabricInstallerSecret `
+                    -encryptionCertificateThumbprint $encryptionCertificateThumbprint
+
+                # Assert
+                $decryptedSecret | Should -eq $fabricInstallerSecret
```

Comment:
Shouldn't the decryptedSecret equal "some secret" not "!!enc!!:some secret". I guess it is just a mock test that returns "!!enc!!:some secret"

Reply:
Yea, it's probably better to have the mock return a different value here